packages/cubejs-databricks-jdbc-driver/src/DatabricksDriver.ts

```diff
@@ -142,6 +142,30 @@ export class DatabricksDriver extends JDBCDriver {
     return !!this.config.readOnly;
   }
 
+  public showDeprecationWarnings() {
```

Comment:
What is the sense of having this method?

Reply:
Deleted ✅